src/config.template.cjs

```diff
@@ -180,6 +180,12 @@ module.exports = {
 	 */
 	host: undefined,
 
+	/**
+	 * Force authentication for API
```

Comment:
Alphanumeric? Dashes? How long?

Reply:
we're using `crypto.randomBytes(24).toString('hex')` now. So the key is 48 characters long, but there are 2^192 possible values.